### PR TITLE
Sign in sign up

### PR DESCRIPTION
**Pull request by mexicanpepe**

### CODE REVIEW

File: features/LoginSignup/Signin.js [hunk 1]
```diff
@@ -1,118 +1,30 @@
-
 import React, {useState} from "react";
 import { TouchableOpacity } from 'react-native';
 import styled from 'styled-components/native'
-import Logo from '../../assets/logo.png'
-import { Firebase_Auth} from '../../FirebaseConfig'
+import Logo from '../../assets/Logo.png'
```
Comment (pnilan) [inline features/LoginSignup/Signin.js]: logo.png path is incorrect

File: features/LoginSignup/Signin.js [hunk 2]
```diff
@@ -125,12 +37,25 @@ const SignIn = () => {
...
-      console.log(response);
-      alert('Sign In Success')
+      console.log(response.user.uid)
+      console.log(response.user.email)
+
+      setProfile({
+        ...profile,
+        'uID': response.user.uid,
+        'email': email,
+        'password': response.user.email,
```
Comment (pnilan) [inline features/LoginSignup/Signin.js]: Can remove this

Comment (pnilan) [reply features/LoginSignup/Signin.js]: And add tokenId

**Review by pnilan — CHANGES_REQUESTED** [review]

**Comment by ahusu** [comment]
I just resolved merged, please pull again before merging!

File: features/LoginSignup/Signin.js (hunk 1 above)
Comment (mexicanpepe) [reply features/LoginSignup/Signin.js]: @pnilan  No its not?

File: features/LoginSignup/Signin.js (hunk 2 above)
Comment (mexicanpepe) [reply features/LoginSignup/Signin.js]: @pnilan Token ID is added

**Pull request merged**